src/templates/base/2019/base.html

```diff
@@ -341,18 +347,37 @@
 {% endmacro %}
 
 {% macro year_switcher(switcher_name) %}
+{# Only show if we currently support multi-year #}
+{% if year_live('2020') %}
```

Comment:
How does this hold up when we have 2021?

I thought the `for y in supported_years` already handled this.

Reply:
`supported_years` is years supported by THIS template. I added an `all_supported_years` variable and generalised this code. Was being a bit lazy so fair enough for calling this out.

Resolved.